src/actions/api/html_conversions/create.cr

```diff
@@ -0,0 +1,9 @@
+class Api::HtmlConversions::Create < ApiAction
+  param input : String = ""
+
+  post "/api/html_conversions" do
+    output = HTML2Lucky::Converter.new(input).convert
+
+    json({content: output}, HTTP::Status::OK)
```

Comment:
I'm not sure, but I think you can do this:

```suggestion
    json({content: output}, :ok)
```

Reply:
Neat!